### PR TITLE
Add db comments for groups_contest_items & contest_participations

### LINKED ISSUE #291: Add comment for `groups_contest_items` + contest_participations tables
Quick task: Could you add a comment for the tables `groups_contest_items` and `contest_participations` you have added recently?

### PR DESCRIPTION
**Pull request by zenovich**
Fixes #291

### CODE REVIEW

File: db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql [hunk 1]
```diff
@@ -6,14 +6,14 @@ CREATE TABLE `groups_contest_items` (
     `can_enter_until` datetime NOT NULL DEFAULT '9999-12-31 23:59:59' COMMENT 'Time until which the group can “enter” this contest',
     `additional_time` time NOT NULL DEFAULT '00:00:00' COMMENT 'Time that was attributed (can be negative) to this group for this contest',
     PRIMARY KEY (`group_id`, `item_id`)
-);
+) COMMENT 'Per-group constraints on contests participation';
```
Comment (smadbe) [inline db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql]: "Group constraints on contest participations"

File: db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql [hunk 2]
```diff
@@ -6,14 +6,14 @@ CREATE TABLE `groups_contest_items` (
...
 
 CREATE TABLE `contest_participations` (
     `group_id` bigint(20) NOT NULL,
     `item_id` bigint(20) NOT NULL,
     `entered_at` datetime DEFAULT NULL COMMENT 'Time at which the group entered the contest',
     PRIMARY KEY (`group_id`, `item_id`)
-);
+) COMMENT 'Connections between contests and contestant groups';
```
Comment (smadbe) [inline db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql]: Not clear what could "connections" be. Relations?
"Team or user information when participating to contests" ? As the fact it is related "end-members" (team or user) is the main differentiator from "contests participation".

File: db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql (hunk 1 above)
Comment (zenovich) [reply db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql]: fixed

File: db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql (hunk 2 above)
Comment (zenovich) [reply db/migrations/51_create_tables_groups_contest_items_and_contest_participations.sql]: fixed

**Pull request merged**